### ISSUE TITLE
Reducing Console Output

### ISSUE THREAD

**Issue by shellscape** [typescript-eslint/typescript-eslint#1225]
**What were you trying to do?**

```json
{
  "parser": "@typescript-eslint/parser",
  "parserOptions": {
    "ecmaFeatures": {
      "generators": false,
      "objectLiteralDuplicateProperties": false
    },
    "ecmaVersion": 9,
    "project": "/tsconfig.json",
    "sourceType": "module"
  },
  "plugins": [
    "@typescript-eslint",
    "import",
    "prettier"
  ],
  "rules": {
    "@typescript-eslint/consistent-type-assertions": "error",
    "@typescript-eslint/consistent-type-definitions": "error",
    "@typescript-eslint/member-ordering": "error",
    "@typescript-eslint/no-inferrable-types": "error",
    "@typescript-eslint/no-unnecessary-type-assertion": "error",
  }
}
```

```ts
// quite literally any TS will do
```

**What did you expect to happen?**

It's not so much an aspect of what I expected to happen, as I didn't have any expectations.

What I was surprised by was an absolute flood of console output when running ESLint with the `@typescript-eslint` plugin.

**What actually happened?**

I'm working in the `rollup/plugins` monorepo and working on a balance for a global ESLint config that'll cover both TS and JS. The config pasted above is a snapshot, sans all of the ESLint rules. When we run linting on say, `scripts/utils`, we end up with a massive amount of console output that (I assume) comes from either the `@typescript-eslint/parser` or the `@typescript-eslint` plugin.

The output mentioned contains phrases such as `CreateProgramWith:` and `Syncing program`, and typically contains a massive list of files in preceded by `root:`. It's two 'console pages' worth of output so I've elected not to sanitize and paste it here, but I can if that's helpful.

What I'm hoping to find by opening this issue, is a means to turn off or greatly reduce the noise. 

**Versions**

| package                                 | version |
| --------------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin`      | `^2.7.0` |
| `@typescript-eslint/parser`             | `^2.7.0` |
| `@typescript-eslint/typescript-estree`  | `n/a` |
| `@typescript-eslint/experimental-utils` | `n/a` |
| `TypeScript`                            | `^3.7.2` |
| `node`                                  | `12.3.1` |
| `npm`                                   | `6.11.2` |

**Comment by bradzacher** [comment]
> What I was surprised by was an absolute flood of console output when running ESLint with the @typescript-eslint plugin

What is this flood? Can you provide an example?
We don't log anything from the plugin - so this statement is objectively false, which implies there's a different source of these logs.
An example would help illustrate what you mean.

**Comment by shellscape** [comment]
Absolutely. (As stated above I didn't want to paste the output given it's length if not needed) Please allow me a few minutes to circle back and sanitize the output, and I'll reply with that content.

**Comment by shellscape** [comment]
I've pasted the entirety of the content output below. It was generated by the config provided above, using this command `$ eslint scripts util`. Within https://github.com/rollup/plugins, the `util` directory contains a lone `test.js` file, which does pass linting. (For context, we're adding our first TypeScript plugin to this repo and attempting a move away from TSLint)

<details>
  <summary>Please click to expand the full output</summary>

Current directory: /root/code/rollup/plugins CaseSensitiveFileNames: false
Synchronizing program
CreatingProgramWith::
  roots: ["/root/code/rollup/plugins/packages/alias/rollup.config.js","/root/code/rollup/plugins/packages/alias/dist/index.es2015.js","/root/code/rollup/plugins/packages/alias/dist/index.js","/root/code/rollup/plugins/packages/alias/src/index.js","/root/code/rollup/plugins/packages/alias/test/test.js","/root/code/rollup/plugins/packages/alias/test/fixtures/aliasMe.js","/root/code/rollup/plugins/packages/alias/test/fixtures/index.js","/root/code/rollup/plugins/packages/alias/test/fixtures/localAliasMe.js","/root/code/rollup/plugins/packages/alias/test/fixtures/nonAliased.js","/root/code/rollup/plugins/packages/alias/test/fixtures/folder/anotherNumber.js","/root/code/rollup/plugins/packages/alias/test/fixtures/folder/hipster.jsx","/root/code/rollup/plugins/packages/auto-install/lib/index.js","/root/code/rollup/plugins/packages/auto-install/test/npm-bare.js","/root/code/rollup/plugins/packages/auto-install/test/npm.js","/root/code/rollup/plugins/packages/auto-install/test/yarn-bare.js","/root/code/rollup/plugins/packages/auto-install/test/yarn.js","/root/code/rollup/plugins/packages/auto-install/test/fixtures/input.js","/root/code/rollup/plugins/packages/beep/lib/index.js","/root/code/rollup/plugins/packages/beep/test/test.js","/root/code/rollup/plugins/packages/beep/test/fixtures/error.config.js","/root/code/rollup/plugins/packages/beep/test/fixtures/error.js","/root/code/rollup/plugins/packages/beep/test/fixtures/pass.config.js","/root/code/rollup/plugins/packages/beep/test/fixtures/pass.js","/root/code/rollup/plugins/packages/beep/test/output/bundle.js","/root/code/rollup/plugins/packages/buble/index.d.ts","/root/code/rollup/plugins/packages/buble/rollup.config.js","/root/code/rollup/plugins/packages/buble/dist/rollup-plugin-buble.cjs.js","/root/code/rollup/plugins/packages/buble/dist/rollup-plugin-buble.es.js","/root/code/rollup/plugins/packages/buble/src/index.js","/root/code/rollup/plugins/packages/buble/test/test.js","/root/code/rollup/plugins/packages/buble/test/types.ts","/root/code/rollup/plugins/packages/buble/test/fixtures/basic/main.js","/root/code/rollup/plugins/packages/dsv/rollup.config.js","/root/code/rollup/plugins/packages/dsv/dist/index.es.js","/root/code/rollup/plugins/packages/dsv/dist/index.js","/root/code/rollup/plugins/packages/dsv/src/index.js","/root/code/rollup/plugins/packages/dsv/test/test.js","/root/code/rollup/plugins/packages/dsv/test/fixtures/basic-csv/main.js","/root/code/rollup/plugins/packages/dsv/test/fixtures/basic-tsv/main.js","/root/code/rollup/plugins/packages/dsv/test/fixtures/process/main.js","/root/code/rollup/plugins/packages/dsv/test/fixtures/process-id/main.js","/root/code/rollup/plugins/packages/inject/index.d.ts","/root/code/rollup/plugins/packages/inject/rollup.config.js","/root/code/rollup/plugins/packages/inject/dist/index.es.js","/root/code/rollup/plugins/packages/inject/dist/index.js","/root/code/rollup/plugins/packages/inject/src/index.js","/root/code/rollup/plugins/packages/inject/test/test.js","/root/code/rollup/plugins/packages/inject/test/types.ts","/root/code/rollup/plugins/packages/inject/test/fixtures/basic/input.js","/root/code/rollup/plugins/packages/inject/test/fixtures/existing/input.js","/root/code/rollup/plugins/packages/inject/test/fixtures/import-namespace/input.js","/root/code/rollup/plugins/packages/inject/test/fixtures/keypaths/input.js","/root/code/rollup/plugins/packages/inject/test/fixtures/keypaths/polyfills/object-assign.js","/root/code/rollup/plugins/packages/inject/test/fixtures/named/input.js","/root/code/rollup/plugins/packages/inject/test/fixtures/non-js/input.js","/root/code/rollup/plugins/packages/inject/test/fixtures/redundant-keys/input.js","/root/code/rollup/plugins/packages/inject/test/fixtures/shadowing/input.js","/root/code/rollup/plugins/packages/inject/test/fixtures/shorthand/input.js","/root/code/rollup/plugins/packages/json/index.d.ts","/root/code/rollup/plugins/packages/json/rollup.config.js","/root/code/rollup/plugins/packages/json/dist/index.es.js","/root/code/rollup/plugins/packages/json/dist/index.js","/root/code/rollup/plugins/packages/json/src/index.js","/root/code/rollup/plugins/packages/json/test/test.js","/root/code/rollup/plugins/packages/json/test/types.ts","/root/code/rollup/plugins/packages/json/test/fixtures/array/main.js","/root/code/rollup/plugins/packages/json/test/fixtures/basic/main.js","/root/code/rollup/plugins/packages/json/test/fixtures/extensionless/main.js","/root/code/rollup/plugins/packages/json/test/fixtures/form/compact.js","/root/code/rollup/plugins/packages/json/test/fixtures/form/customIndent.js","/root/code/rollup/plugins/packages/json/test/fixtures/form/default.js","/root/code/rollup/plugins/packages/json/test/fixtures/form/namedExports.js","/root/code/rollup/plugins/packages/json/test/fixtures/form/preferConst.js","/root/code/rollup/plugins/packages/json/test/fixtures/garbage/main.js","/root/code/rollup/plugins/packages/json/test/fixtures/literal/main.js","/root/code/rollup/plugins/packages/json/test/fixtures/named/main.js","/root/code/rollup/plugins/packages/json/test/fixtures/no-valid-keys/main.js","/root/code/rollup/plugins/packages/replace/index.d.ts","/root/code/rollup/plugins/packages/replace/rollup.config.js","/root/code/rollup/plugins/packages/replace/dist/rollup-plugin-replace.cjs.js","/root/code/rollup/plugins/packages/replace/dist/rollup-plugin-replace.es.js","/root/code/rollup/plugins/packages/replace/src/index.js","/root/code/rollup/plugins/packages/replace/test/form.js","/root/code/rollup/plugins/packages/replace/test/function.js","/root/code/rollup/plugins/packages/replace/test/misc.js","/root/code/rollup/plugins/packages/replace/test/types.ts","/root/code/rollup/plugins/packages/replace/test/fixtures/form/delimiters/_config.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/delimiters/input.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/delimiters/output.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/match-variables/_config.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/match-variables/input.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/match-variables/output.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/observe-plugin-options/_config.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/observe-plugin-options/input.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/observe-plugin-options/output.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/replace-nothing/_config.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/replace-nothing/input.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/replace-nothing/output.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/replace-strings/_config.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/replace-strings/input.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/replace-strings/output.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/replacement-function/_config.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/replacement-function/input.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/replacement-function/output.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/special-characters/_config.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/special-characters/input.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/special-characters/output.js","/root/code/rollup/plugins/packages/replace/test/fixtures/function/replacement-function/_config.js","/root/code/rollup/plugins/packages/replace/test/fixtures/function/replacement-function/main.js","/root/code/rollup/plugins/packages/replace/test/fixtures/function/replacement-function/dir/foo.js","/root/code/rollup/plugins/packages/replace/test/fixtures/function/word-boundaries/_config.js","/root/code/rollup/plugins/packages/replace/test/fixtures/function/word-boundaries/main.js","/root/code/rollup/plugins/packages/replace/test/helpers/util.js","/root/code/rollup/plugins/packages/strip/rollup.config.js","/root/code/rollup/plugins/packages/strip/dist/index.es.js","/root/code/rollup/plugins/packages/strip/dist/index.js","/root/code/rollup/plugins/packages/strip/src/index.js","/root/code/rollup/plugins/packages/strip/test/test.js","/root/code/rollup/plugins/packages/strip/test/fixtures/assert/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/console/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/console-custom/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/custom/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/debugger/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/debugger-false/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/if-block/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/inline-call-expressions/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/inline-if/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/inline-while/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/label-block/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/label-block-discriminate/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/label-block-multiple/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/object-destructuring-default/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/super-method/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/switch-case/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/this-method/input.js","/root/code/rollup/plugins/packages/utils/rollup.config.js","/root/code/rollup/plugins/packages/utils/dist/index.es.js","/root/code/rollup/plugins/packages/utils/dist/index.js","/root/code/rollup/plugins/packages/utils/src/index.ts","/root/code/rollup/plugins/packages/utils/src/plugins/addExtension.ts","/root/code/rollup/plugins/packages/utils/src/plugins/attachScopes.ts","/root/code/rollup/plugins/packages/utils/src/plugins/createFilter.ts","/root/code/rollup/plugins/packages/utils/src/plugins/dataToEsm.ts","/root/code/rollup/plugins/packages/utils/src/plugins/extractAssignedNames.ts","/root/code/rollup/plugins/packages/utils/src/plugins/makeLegalIdentifier.ts","/root/code/rollup/plugins/packages/utils/src/utils/ensureArray.ts","/root/code/rollup/plugins/packages/utils/test/addExtension.test.ts","/root/code/rollup/plugins/packages/utils/test/attachScopes.test.ts","/root/code/rollup/plugins/packages/utils/test/createFilter.test.ts","/root/code/rollup/plugins/packages/utils/test/dataToEsm.test.ts","/root/code/rollup/plugins/packages/utils/test/extractAssignedNames.test.ts","/root/code/rollup/plugins/packages/utils/test/makeLegalIdentifier.test.ts","/root/code/rollup/plugins/packages/utils/types/index.d.ts","/root/code/rollup/plugins/packages/wasm/rollup.config.js","/root/code/rollup/plugins/packages/wasm/dist/index.js","/root/code/rollup/plugins/packages/wasm/src/index.js","/root/code/rollup/plugins/packages/wasm/test/index.js","/root/code/rollup/plugins/packages/wasm/test/fixtures/async.js","/root/code/rollup/plugins/packages/wasm/test/fixtures/complex.js","/root/code/rollup/plugins/packages/wasm/test/fixtures/imports.js","/root/code/rollup/plugins/packages/wasm/test/fixtures/sync.js","/root/code/rollup/plugins/packages/yaml/rollup.config.js","/root/code/rollup/plugins/packages/yaml/dist/index.es.js","/root/code/rollup/plugins/packages/yaml/dist/index.js","/root/code/rollup/plugins/packages/yaml/src/index.js","/root/code/rollup/plugins/packages/yaml/test/test.js","/root/code/rollup/plugins/packages/yaml/test/fixtures/basic/main.js","/root/code/rollup/plugins/packages/yaml/test/fixtures/extensionless/main.js","/root/code/rollup/plugins/packages/yaml/test/fixtures/named/main.js","/root/code/rollup/plugins/packages/yaml/test/fixtures/transform/main.js","/root/code/rollup/plugins/packages/yaml/test/fixtures/yml/main.js","/root/code/rollup/plugins/scripts/codecov.js","/root/code/rollup/plugins/scripts/disallow-npm.js","/root/code/rollup/plugins/scripts/publish.js","/root/code/rollup/plugins/scripts/run-changed.js","/root/code/rollup/plugins/util/test.js"]
  options: {"allowSyntheticDefaultImports":true,"diagnostics":true,"esModuleInterop":true,"lib":["lib.es2015.d.ts","lib.dom.d.ts"],"module":1,"noEmitOnError":true,"noUnusedLocals":true,"noUnusedParameters":true,"pretty":true,"skipLibCheck":true,"sourceMap":true,"strict":true,"target":2,"allowNonTsExtensions":true,"allowJs":true,"checkJs":true,"noEmit":true,"configFilePath":"/root/code/rollup/plugins/tsconfig.json"}
Synchronizing program
Synchronizing program
FileWatcher:: Triggered with /root/code/rollup/plugins/scripts/publish.js 1:: WatchInfo: /root/code/rollup/plugins/scripts/publish.js 250 Source file
Scheduling update
Synchronizing program
CreatingProgramWith::
  roots: ["/root/code/rollup/plugins/packages/alias/rollup.config.js","/root/code/rollup/plugins/packages/alias/dist/index.es2015.js","/root/code/rollup/plugins/packages/alias/dist/index.js","/root/code/rollup/plugins/packages/alias/src/index.js","/root/code/rollup/plugins/packages/alias/test/test.js","/root/code/rollup/plugins/packages/alias/test/fixtures/aliasMe.js","/root/code/rollup/plugins/packages/alias/test/fixtures/index.js","/root/code/rollup/plugins/packages/alias/test/fixtures/localAliasMe.js","/root/code/rollup/plugins/packages/alias/test/fixtures/nonAliased.js","/root/code/rollup/plugins/packages/alias/test/fixtures/folder/anotherNumber.js","/root/code/rollup/plugins/packages/alias/test/fixtures/folder/hipster.jsx","/root/code/rollup/plugins/packages/auto-install/lib/index.js","/root/code/rollup/plugins/packages/auto-install/test/npm-bare.js","/root/code/rollup/plugins/packages/auto-install/test/npm.js","/root/code/rollup/plugins/packages/auto-install/test/yarn-bare.js","/root/code/rollup/plugins/packages/auto-install/test/yarn.js","/root/code/rollup/plugins/packages/auto-install/test/fixtures/input.js","/root/code/rollup/plugins/packages/beep/lib/index.js","/root/code/rollup/plugins/packages/beep/test/test.js","/root/code/rollup/plugins/packages/beep/test/fixtures/error.config.js","/root/code/rollup/plugins/packages/beep/test/fixtures/error.js","/root/code/rollup/plugins/packages/beep/test/fixtures/pass.config.js","/root/code/rollup/plugins/packages/beep/test/fixtures/pass.js","/root/code/rollup/plugins/packages/beep/test/output/bundle.js","/root/code/rollup/plugins/packages/buble/index.d.ts","/root/code/rollup/plugins/packages/buble/rollup.config.js","/root/code/rollup/plugins/packages/buble/dist/rollup-plugin-buble.cjs.js","/root/code/rollup/plugins/packages/buble/dist/rollup-plugin-buble.es.js","/root/code/rollup/plugins/packages/buble/src/index.js","/root/code/rollup/plugins/packages/buble/test/test.js","/root/code/rollup/plugins/packages/buble/test/types.ts","/root/code/rollup/plugins/packages/buble/test/fixtures/basic/main.js","/root/code/rollup/plugins/packages/dsv/rollup.config.js","/root/code/rollup/plugins/packages/dsv/dist/index.es.js","/root/code/rollup/plugins/packages/dsv/dist/index.js","/root/code/rollup/plugins/packages/dsv/src/index.js","/root/code/rollup/plugins/packages/dsv/test/test.js","/root/code/rollup/plugins/packages/dsv/test/fixtures/basic-csv/main.js","/root/code/rollup/plugins/packages/dsv/test/fixtures/basic-tsv/main.js","/root/code/rollup/plugins/packages/dsv/test/fixtures/process/main.js","/root/code/rollup/plugins/packages/dsv/test/fixtures/process-id/main.js","/root/code/rollup/plugins/packages/inject/index.d.ts","/root/code/rollup/plugins/packages/inject/rollup.config.js","/root/code/rollup/plugins/packages/inject/dist/index.es.js","/root/code/rollup/plugins/packages/inject/dist/index.js","/root/code/rollup/plugins/packages/inject/src/index.js","/root/code/rollup/plugins/packages/inject/test/test.js","/root/code/rollup/plugins/packages/inject/test/types.ts","/root/code/rollup/plugins/packages/inject/test/fixtures/basic/input.js","/root/code/rollup/plugins/packages/inject/test/fixtures/existing/input.js","/root/code/rollup/plugins/packages/inject/test/fixtures/import-namespace/input.js","/root/code/rollup/plugins/packages/inject/test/fixtures/keypaths/input.js","/root/code/rollup/plugins/packages/inject/test/fixtures/keypaths/polyfills/object-assign.js","/root/code/rollup/plugins/packages/inject/test/fixtures/named/input.js","/root/code/rollup/plugins/packages/inject/test/fixtures/non-js/input.js","/root/code/rollup/plugins/packages/inject/test/fixtures/redundant-keys/input.js","/root/code/rollup/plugins/packages/inject/test/fixtures/shadowing/input.js","/root/code/rollup/plugins/packages/inject/test/fixtures/shorthand/input.js","/root/code/rollup/plugins/packages/json/index.d.ts","/root/code/rollup/plugins/packages/json/rollup.config.js","/root/code/rollup/plugins/packages/json/dist/index.es.js","/root/code/rollup/plugins/packages/json/dist/index.js","/root/code/rollup/plugins/packages/json/src/index.js","/root/code/rollup/plugins/packages/json/test/test.js","/root/code/rollup/plugins/packages/json/test/types.ts","/root/code/rollup/plugins/packages/json/test/fixtures/array/main.js","/root/code/rollup/plugins/packages/json/test/fixtures/basic/main.js","/root/code/rollup/plugins/packages/json/test/fixtures/extensionless/main.js","/root/code/rollup/plugins/packages/json/test/fixtures/form/compact.js","/root/code/rollup/plugins/packages/json/test/fixtures/form/customIndent.js","/root/code/rollup/plugins/packages/json/test/fixtures/form/default.js","/root/code/rollup/plugins/packages/json/test/fixtures/form/namedExports.js","/root/code/rollup/plugins/packages/json/test/fixtures/form/preferConst.js","/root/code/rollup/plugins/packages/json/test/fixtures/garbage/main.js","/root/code/rollup/plugins/packages/json/test/fixtures/literal/main.js","/root/code/rollup/plugins/packages/json/test/fixtures/named/main.js","/root/code/rollup/plugins/packages/json/test/fixtures/no-valid-keys/main.js","/root/code/rollup/plugins/packages/replace/index.d.ts","/root/code/rollup/plugins/packages/replace/rollup.config.js","/root/code/rollup/plugins/packages/replace/dist/rollup-plugin-replace.cjs.js","/root/code/rollup/plugins/packages/replace/dist/rollup-plugin-replace.es.js","/root/code/rollup/plugins/packages/replace/src/index.js","/root/code/rollup/plugins/packages/replace/test/form.js","/root/code/rollup/plugins/packages/replace/test/function.js","/root/code/rollup/plugins/packages/replace/test/misc.js","/root/code/rollup/plugins/packages/replace/test/types.ts","/root/code/rollup/plugins/packages/replace/test/fixtures/form/delimiters/_config.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/delimiters/input.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/delimiters/output.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/match-variables/_config.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/match-variables/input.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/match-variables/output.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/observe-plugin-options/_config.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/observe-plugin-options/input.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/observe-plugin-options/output.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/replace-nothing/_config.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/replace-nothing/input.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/replace-nothing/output.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/replace-strings/_config.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/replace-strings/input.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/replace-strings/output.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/replacement-function/_config.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/replacement-function/input.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/replacement-function/output.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/special-characters/_config.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/special-characters/input.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/special-characters/output.js","/root/code/rollup/plugins/packages/replace/test/fixtures/function/replacement-function/_config.js","/root/code/rollup/plugins/packages/replace/test/fixtures/function/replacement-function/main.js","/root/code/rollup/plugins/packages/replace/test/fixtures/function/replacement-function/dir/foo.js","/root/code/rollup/plugins/packages/replace/test/fixtures/function/word-boundaries/_config.js","/root/code/rollup/plugins/packages/replace/test/fixtures/function/word-boundaries/main.js","/root/code/rollup/plugins/packages/replace/test/helpers/util.js","/root/code/rollup/plugins/packages/strip/rollup.config.js","/root/code/rollup/plugins/packages/strip/dist/index.es.js","/root/code/rollup/plugins/packages/strip/dist/index.js","/root/code/rollup/plugins/packages/strip/src/index.js","/root/code/rollup/plugins/packages/strip/test/test.js","/root/code/rollup/plugins/packages/strip/test/fixtures/assert/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/console/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/console-custom/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/custom/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/debugger/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/debugger-false/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/if-block/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/inline-call-expressions/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/inline-if/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/inline-while/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/label-block/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/label-block-discriminate/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/label-block-multiple/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/object-destructuring-default/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/super-method/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/switch-case/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/this-method/input.js","/root/code/rollup/plugins/packages/utils/rollup.config.js","/root/code/rollup/plugins/packages/utils/dist/index.es.js","/root/code/rollup/plugins/packages/utils/dist/index.js","/root/code/rollup/plugins/packages/utils/src/index.ts","/root/code/rollup/plugins/packages/utils/src/plugins/addExtension.ts","/root/code/rollup/plugins/packages/utils/src/plugins/attachScopes.ts","/root/code/rollup/plugins/packages/utils/src/plugins/createFilter.ts","/root/code/rollup/plugins/packages/utils/src/plugins/dataToEsm.ts","/root/code/rollup/plugins/packages/utils/src/plugins/extractAssignedNames.ts","/root/code/rollup/plugins/packages/utils/src/plugins/makeLegalIdentifier.ts","/root/code/rollup/plugins/packages/utils/src/utils/ensureArray.ts","/root/code/rollup/plugins/packages/utils/test/addExtension.test.ts","/root/code/rollup/plugins/packages/utils/test/attachScopes.test.ts","/root/code/rollup/plugins/packages/utils/test/createFilter.test.ts","/root/code/rollup/plugins/packages/utils/test/dataToEsm.test.ts","/root/code/rollup/plugins/packages/utils/test/extractAssignedNames.test.ts","/root/code/rollup/plugins/packages/utils/test/makeLegalIdentifier.test.ts","/root/code/rollup/plugins/packages/utils/types/index.d.ts","/root/code/rollup/plugins/packages/wasm/rollup.config.js","/root/code/rollup/plugins/packages/wasm/dist/index.js","/root/code/rollup/plugins/packages/wasm/src/index.js","/root/code/rollup/plugins/packages/wasm/test/index.js","/root/code/rollup/plugins/packages/wasm/test/fixtures/async.js","/root/code/rollup/plugins/packages/wasm/test/fixtures/complex.js","/root/code/rollup/plugins/packages/wasm/test/fixtures/imports.js","/root/code/rollup/plugins/packages/wasm/test/fixtures/sync.js","/root/code/rollup/plugins/packages/yaml/rollup.config.js","/root/code/rollup/plugins/packages/yaml/dist/index.es.js","/root/code/rollup/plugins/packages/yaml/dist/index.js","/root/code/rollup/plugins/packages/yaml/src/index.js","/root/code/rollup/plugins/packages/yaml/test/test.js","/root/code/rollup/plugins/packages/yaml/test/fixtures/basic/main.js","/root/code/rollup/plugins/packages/yaml/test/fixtures/extensionless/main.js","/root/code/rollup/plugins/packages/yaml/test/fixtures/named/main.js","/root/code/rollup/plugins/packages/yaml/test/fixtures/transform/main.js","/root/code/rollup/plugins/packages/yaml/test/fixtures/yml/main.js","/root/code/rollup/plugins/scripts/codecov.js","/root/code/rollup/plugins/scripts/disallow-npm.js","/root/code/rollup/plugins/scripts/publish.js","/root/code/rollup/plugins/scripts/run-changed.js","/root/code/rollup/plugins/util/test.js"]
  options: {"allowSyntheticDefaultImports":true,"diagnostics":true,"esModuleInterop":true,"lib":["lib.es2015.d.ts","lib.dom.d.ts"],"module":1,"noEmitOnError":true,"noUnusedLocals":true,"noUnusedParameters":true,"pretty":true,"skipLibCheck":true,"sourceMap":true,"strict":true,"target":2,"allowNonTsExtensions":true,"allowJs":true,"checkJs":true,"noEmit":true,"configFilePath":"/root/code/rollup/plugins/tsconfig.json"}
Elapsed:: 17ms FileWatcher:: Triggered with /root/code/rollup/plugins/scripts/publish.js 1:: WatchInfo: /root/code/rollup/plugins/scripts/publish.js 250 Source file
Synchronizing program
FileWatcher:: Triggered with /root/code/rollup/plugins/scripts/run-changed.js 1:: WatchInfo: /root/code/rollup/plugins/scripts/run-changed.js 250 Source file
Scheduling update
Synchronizing program
CreatingProgramWith::
  roots: ["/root/code/rollup/plugins/packages/alias/rollup.config.js","/root/code/rollup/plugins/packages/alias/dist/index.es2015.js","/root/code/rollup/plugins/packages/alias/dist/index.js","/root/code/rollup/plugins/packages/alias/src/index.js","/root/code/rollup/plugins/packages/alias/test/test.js","/root/code/rollup/plugins/packages/alias/test/fixtures/aliasMe.js","/root/code/rollup/plugins/packages/alias/test/fixtures/index.js","/root/code/rollup/plugins/packages/alias/test/fixtures/localAliasMe.js","/root/code/rollup/plugins/packages/alias/test/fixtures/nonAliased.js","/root/code/rollup/plugins/packages/alias/test/fixtures/folder/anotherNumber.js","/root/code/rollup/plugins/packages/alias/test/fixtures/folder/hipster.jsx","/root/code/rollup/plugins/packages/auto-install/lib/index.js","/root/code/rollup/plugins/packages/auto-install/test/npm-bare.js","/root/code/rollup/plugins/packages/auto-install/test/npm.js","/root/code/rollup/plugins/packages/auto-install/test/yarn-bare.js","/root/code/rollup/plugins/packages/auto-install/test/yarn.js","/root/code/rollup/plugins/packages/auto-install/test/fixtures/input.js","/root/code/rollup/plugins/packages/beep/lib/index.js","/root/code/rollup/plugins/packages/beep/test/test.js","/root/code/rollup/plugins/packages/beep/test/fixtures/error.config.js","/root/code/rollup/plugins/packages/beep/test/fixtures/error.js","/root/code/rollup/plugins/packages/beep/test/fixtures/pass.config.js","/root/code/rollup/plugins/packages/beep/test/fixtures/pass.js","/root/code/rollup/plugins/packages/beep/test/output/bundle.js","/root/code/rollup/plugins/packages/buble/index.d.ts","/root/code/rollup/plugins/packages/buble/rollup.config.js","/root/code/rollup/plugins/packages/buble/dist/rollup-plugin-buble.cjs.js","/root/code/rollup/plugins/packages/buble/dist/rollup-plugin-buble.es.js","/root/code/rollup/plugins/packages/buble/src/index.js","/root/code/rollup/plugins/packages/buble/test/test.js","/root/code/rollup/plugins/packages/buble/test/types.ts","/root/code/rollup/plugins/packages/buble/test/fixtures/basic/main.js","/root/code/rollup/plugins/packages/dsv/rollup.config.js","/root/code/rollup/plugins/packages/dsv/dist/index.es.js","/root/code/rollup/plugins/packages/dsv/dist/index.js","/root/code/rollup/plugins/packages/dsv/src/index.js","/root/code/rollup/plugins/packages/dsv/test/test.js","/root/code/rollup/plugins/packages/dsv/test/fixtures/basic-csv/main.js","/root/code/rollup/plugins/packages/dsv/test/fixtures/basic-tsv/main.js","/root/code/rollup/plugins/packages/dsv/test/fixtures/process/main.js","/root/code/rollup/plugins/packages/dsv/test/fixtures/process-id/main.js","/root/code/rollup/plugins/packages/inject/index.d.ts","/root/code/rollup/plugins/packages/inject/rollup.config.js","/root/code/rollup/plugins/packages/inject/dist/index.es.js","/root/code/rollup/plugins/packages/inject/dist/index.js","/root/code/rollup/plugins/packages/inject/src/index.js","/root/code/rollup/plugins/packages/inject/test/test.js","/root/code/rollup/plugins/packages/inject/test/types.ts","/root/code/rollup/plugins/packages/inject/test/fixtures/basic/input.js","/root/code/rollup/plugins/packages/inject/test/fixtures/existing/input.js","/root/code/rollup/plugins/packages/inject/test/fixtures/import-namespace/input.js","/root/code/rollup/plugins/packages/inject/test/fixtures/keypaths/input.js","/root/code/rollup/plugins/packages/inject/test/fixtures/keypaths/polyfills/object-assign.js","/root/code/rollup/plugins/packages/inject/test/fixtures/named/input.js","/root/code/rollup/plugins/packages/inject/test/fixtures/non-js/input.js","/root/code/rollup/plugins/packages/inject/test/fixtures/redundant-keys/input.js","/root/code/rollup/plugins/packages/inject/test/fixtures/shadowing/input.js","/root/code/rollup/plugins/packages/inject/test/fixtures/shorthand/input.js","/root/code/rollup/plugins/packages/json/index.d.ts","/root/code/rollup/plugins/packages/json/rollup.config.js","/root/code/rollup/plugins/packages/json/dist/index.es.js","/root/code/rollup/plugins/packages/json/dist/index.js","/root/code/rollup/plugins/packages/json/src/index.js","/root/code/rollup/plugins/packages/json/test/test.js","/root/code/rollup/plugins/packages/json/test/types.ts","/root/code/rollup/plugins/packages/json/test/fixtures/array/main.js","/root/code/rollup/plugins/packages/json/test/fixtures/basic/main.js","/root/code/rollup/plugins/packages/json/test/fixtures/extensionless/main.js","/root/code/rollup/plugins/packages/json/test/fixtures/form/compact.js","/root/code/rollup/plugins/packages/json/test/fixtures/form/customIndent.js","/root/code/rollup/plugins/packages/json/test/fixtures/form/default.js","/root/code/rollup/plugins/packages/json/test/fixtures/form/namedExports.js","/root/code/rollup/plugins/packages/json/test/fixtures/form/preferConst.js","/root/code/rollup/plugins/packages/json/test/fixtures/garbage/main.js","/root/code/rollup/plugins/packages/json/test/fixtures/literal/main.js","/root/code/rollup/plugins/packages/json/test/fixtures/named/main.js","/root/code/rollup/plugins/packages/json/test/fixtures/no-valid-keys/main.js","/root/code/rollup/plugins/packages/replace/index.d.ts","/root/code/rollup/plugins/packages/replace/rollup.config.js","/root/code/rollup/plugins/packages/replace/dist/rollup-plugin-replace.cjs.js","/root/code/rollup/plugins/packages/replace/dist/rollup-plugin-replace.es.js","/root/code/rollup/plugins/packages/replace/src/index.js","/root/code/rollup/plugins/packages/replace/test/form.js","/root/code/rollup/plugins/packages/replace/test/function.js","/root/code/rollup/plugins/packages/replace/test/misc.js","/root/code/rollup/plugins/packages/replace/test/types.ts","/root/code/rollup/plugins/packages/replace/test/fixtures/form/delimiters/_config.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/delimiters/input.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/delimiters/output.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/match-variables/_config.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/match-variables/input.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/match-variables/output.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/observe-plugin-options/_config.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/observe-plugin-options/input.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/observe-plugin-options/output.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/replace-nothing/_config.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/replace-nothing/input.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/replace-nothing/output.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/replace-strings/_config.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/replace-strings/input.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/replace-strings/output.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/replacement-function/_config.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/replacement-function/input.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/replacement-function/output.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/special-characters/_config.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/special-characters/input.js","/root/code/rollup/plugins/packages/replace/test/fixtures/form/special-characters/output.js","/root/code/rollup/plugins/packages/replace/test/fixtures/function/replacement-function/_config.js","/root/code/rollup/plugins/packages/replace/test/fixtures/function/replacement-function/main.js","/root/code/rollup/plugins/packages/replace/test/fixtures/function/replacement-function/dir/foo.js","/root/code/rollup/plugins/packages/replace/test/fixtures/function/word-boundaries/_config.js","/root/code/rollup/plugins/packages/replace/test/fixtures/function/word-boundaries/main.js","/root/code/rollup/plugins/packages/replace/test/helpers/util.js","/root/code/rollup/plugins/packages/strip/rollup.config.js","/root/code/rollup/plugins/packages/strip/dist/index.es.js","/root/code/rollup/plugins/packages/strip/dist/index.js","/root/code/rollup/plugins/packages/strip/src/index.js","/root/code/rollup/plugins/packages/strip/test/test.js","/root/code/rollup/plugins/packages/strip/test/fixtures/assert/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/console/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/console-custom/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/custom/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/debugger/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/debugger-false/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/if-block/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/inline-call-expressions/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/inline-if/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/inline-while/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/label-block/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/label-block-discriminate/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/label-block-multiple/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/object-destructuring-default/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/super-method/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/switch-case/input.js","/root/code/rollup/plugins/packages/strip/test/fixtures/this-method/input.js","/root/code/rollup/plugins/packages/utils/rollup.config.js","/root/code/rollup/plugins/packages/utils/dist/index.es.js","/root/code/rollup/plugins/packages/utils/dist/index.js","/root/code/rollup/plugins/packages/utils/src/index.ts","/root/code/rollup/plugins/packages/utils/src/plugins/addExtension.ts","/root/code/rollup/plugins/packages/utils/src/plugins/attachScopes.ts","/root/code/rollup/plugins/packages/utils/src/plugins/createFilter.ts","/root/code/rollup/plugins/packages/utils/src/plugins/dataToEsm.ts","/root/code/rollup/plugins/packages/utils/src/plugins/extractAssignedNames.ts","/root/code/rollup/plugins/packages/utils/src/plugins/makeLegalIdentifier.ts","/root/code/rollup/plugins/packages/utils/src/utils/ensureArray.ts","/root/code/rollup/plugins/packages/utils/test/addExtension.test.ts","/root/code/rollup/plugins/packages/utils/test/attachScopes.test.ts","/root/code/rollup/plugins/packages/utils/test/createFilter.test.ts","/root/code/rollup/plugins/packages/utils/test/dataToEsm.test.ts","/root/code/rollup/plugins/packages/utils/test/extractAssignedNames.test.ts","/root/code/rollup/plugins/packages/utils/test/makeLegalIdentifier.test.ts","/root/code/rollup/plugins/packages/utils/types/index.d.ts","/root/code/rollup/plugins/packages/wasm/rollup.config.js","/root/code/rollup/plugins/packages/wasm/dist/index.js","/root/code/rollup/plugins/packages/wasm/src/index.js","/root/code/rollup/plugins/packages/wasm/test/index.js","/root/code/rollup/plugins/packages/wasm/test/fixtures/async.js","/root/code/rollup/plugins/packages/wasm/test/fixtures/complex.js","/root/code/rollup/plugins/packages/wasm/test/fixtures/imports.js","/root/code/rollup/plugins/packages/wasm/test/fixtures/sync.js","/root/code/rollup/plugins/packages/yaml/rollup.config.js","/root/code/rollup/plugins/packages/yaml/dist/index.es.js","/root/code/rollup/plugins/packages/yaml/dist/index.js","/root/code/rollup/plugins/packages/yaml/src/index.js","/root/code/rollup/plugins/packages/yaml/test/test.js","/root/code/rollup/plugins/packages/yaml/test/fixtures/basic/main.js","/root/code/rollup/plugins/packages/yaml/test/fixtures/extensionless/main.js","/root/code/rollup/plugins/packages/yaml/test/fixtures/named/main.js","/root/code/rollup/plugins/packages/yaml/test/fixtures/transform/main.js","/root/code/rollup/plugins/packages/yaml/test/fixtures/yml/main.js","/root/code/rollup/plugins/scripts/codecov.js","/root/code/rollup/plugins/scripts/disallow-npm.js","/root/code/rollup/plugins/scripts/publish.js","/root/code/rollup/plugins/scripts/run-changed.js","/root/code/rollup/plugins/util/test.js"]
  options: {"allowSyntheticDefaultImports":true,"diagnostics":true,"esModuleInterop":true,"lib":["lib.es2015.d.ts","lib.dom.d.ts"],"module":1,"noEmitOnError":true,"noUnusedLocals":true,"noUnusedParameters":true,"pretty":true,"skipLibCheck":true,"sourceMap":true,"strict":true,"target":2,"allowNonTsExtensions":true,"allowJs":true,"checkJs":true,"noEmit":true,"configFilePath":"/root/code/rollup/plugins/tsconfig.json"}
Elapsed:: 16ms FileWatcher:: Triggered with /root/code/rollup/plugins/scripts/run-changed.js 1:: WatchInfo: /root/code/rollup/plugins/scripts/run-changed.js 250 Source file
Synchronizing program
Synchronizing program
</details>

**Comment by shellscape** [comment]
Eureka! I've keep at triaging this after the issue was opened and it looks like a rogue `"diagnostics": true,` in an extended `tsconfig.json` is the culprit. Hopefully this will help someone searching in a similar situation in the future.

**Comment by bradzacher** [comment]
> looks like a rogue "diagnostics": true, in an extended tsconfig.json is the culprit

This is what I expected when I saw `CreateProgramWith` and `Syncing program`, but I wanted to see some more logs before making judgement.

Glad you found it!

**Comment by AnyhowStep** [comment]
Why would setting `diagnostics` to `true` cause all that output with typescript-eslint?
`tsc` itself doesn't output all that stuff when I run it.

**Comment by bradzacher** [comment]
> Why would setting diagnostics to true cause all that output with typescript-eslint?

It doesn't, it causes output from typescript itself.

`diagnostics` and `extendedDiagnostics` are typescript compiler options.
https://www.typescriptlang.org/docs/handbook/compiler-options.html

If you turn it on, and compile using that tsconfig, it will cause typescript to log a  bunch of extra stuff.

Nothing to do with our project at all - we just pass through the tsconfig to typescript without touching it.
We only log stuff ourselves if you use the `DEBUG=typescript-eslint:*` environment variable, or pass `debugLevel` as a parser option.

I don't know anything about your setup, so it's hard to provide guidance.
This is why our contributing guideles explicitly ask you don't comment on old threads.
https://github.com/typescript-eslint/typescript-eslint/blob/master/CONTRIBUTING.md#commenting
> Please refrain from commenting on old, closed issues and PRs. Your issue is rarely related enough to a closed issue to warrant "necroing" a dead thread - raising a new issue means you can fill in the template, and make it easier for us to help you.
